.travis.yml

```diff
@@ -8,16 +8,40 @@ go:
   - 1.5.3
   - 1.6
 
+env:
+  - TARGET=amd64
+  - TARGET=arm64
+
+matrix:
+  exclude:
+    - go: 1.6
```

Comment:
No, I thought to put it in to speed up the travis job.  I'll remove this exclude.